src/lib/rest/rest.cpp

```diff
@@ -240,7 +240,7 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
   {
     containsForbiddenChars = forbiddenChars(val, ";");
   }
-  else if ((key != "q") && (key != "idPattern"))
+  else if ((key != "q") && (key != "mq") && (key != "idPattern"))
```

Comment:
In line 210 you are using URI_PARAM_Q and URI_PARAM_MQ but here you are using "q" and "mq"... Is there a reason for that? Or just a pending refactor to align existing "q", "mq", etc. to their respective URI_PARAM_* ?